scripts/gen_dlaf_weak-gpu.py

```diff
@@ -18,17 +18,27 @@
 import miniapps as mp
 import systems
 
-system = systems.cscs["daint-mc"]
+system = systems.cscs["daint-gpu"]
 
-dlafpath = "<path_to_dlaf>"
+dlafpath = "<path_to_dlaf_build_dir>"
 
 run_dir = f"~/ws/runs"
```

Comment:
having the same default directory for strong and weak is dangerous.

Reply:
See https://github.com/eth-cscs/DLA-Future/pull/840/commits/3300d59a66caa3a932078adb831100d72d78cc29